node/pod.go

```diff
@@ -247,7 +252,17 @@ func (pc *PodController) enqueuePodStatusUpdate(ctx context.Context, q workqueue
 			kpod.lastPodStatusReceivedFromProvider = pod
 			kpod.Unlock()
 			q.AddRateLimited(key)
+			return
+		}
+		// check the pod from lister
+		_, err := pc.podsLister.Pods(pod.Namespace).Get(pod.Name)
+		if err != nil {
```

Comment:
This should be attenuated to not found.

Reply:
Err. This reads cache right? So, the only valid error should be not found or return.

Reply:
If the error is not `Not Found` what we should do?

Reply:
Good question. In the multi-queue model, I suggest at a minimum, we log it.

Reply:
IMO， just log it